sdk/appplatform/arm-appplatform/samples/v3/javascript/apiPortalCustomDomainsCreateOrUpdateSample.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+const { AppPlatformManagementClient } = require("@azure/arm-appplatform");
```

Comment:
delete [v3-beta](https://github.com/Azure/azure-sdk-for-js/tree/026cd2c1bae87017f76b9114020666522e8b99e8/sdk/appplatform/arm-appplatform/samples/v3-beta)